boot/bootutil/src/image_rsa.c

```diff
@@ -170,22 +170,19 @@ bootutil_cmp_rsasig(mbedtls_rsa_context *ctx, uint8_t *hash, uint32_t hlen,
     uint8_t db_mask[PSS_MASK_LEN];
     uint8_t h2[PSS_HLEN];
     int i;
-    int rc = 0;
-    fih_int fih_rc = FIH_FAILURE;
+    fih_ret fih_rc;
+    FIH_SET(fih_rc, FIH_FAILURE);
```

Comment:
Any reason this is set separately with an FIH_SET instead of just assigned like the other uses seem to be?